src/pykiso/lib/connectors/cc_pcan_can.py

```diff
@@ -84,6 +87,7 @@ def __init__(
         :param channel: the can interface name
         :param state: BusState of the channel
         :param trace_path: path to write the trace
+        :param trace_name: name of the trace, must be of the .trc format
```

Comment:
Wouldn't it make more sense to include the trace name in the trace path, which could then be either the path to a folder or to a file?

Reply:
Done